src/Event/Productions/EventCannotBeAddedToProduction.php

```diff
@@ -9,7 +9,14 @@ final class EventCannotBeAddedToProduction extends Exception
     public static function becauseItAlreadyBelongsToAnotherProduction(string $eventId, ProductionId $productionId)
     {
         return new self(
-            'Event with id ' . $eventId . ' can not be added to production with id ' . $productionId->toNative()
+            'Event with id ' . $eventId . ' cannot be added to production with id ' . $productionId->toNative()
+        );
+    }
+
+    public static function becauseTheyAlreadyBelongToAnotherProduction(array $eventIds, ProductionId $productionId)
+    {
+        return new self(
+            'Events with id ' . join(',', $eventIds) . ' cannot be added to production with id ' . $productionId->toNative()
```

Comment:
Nit: Should it be `Events with ids` (plural)?

Also, should we add an explanation at the end cfr the method names? E.g. `Events with ids ... cannot be added to production with id ... because they already belong to another production`?

Reply:
Fixed

Reply:
Sorry to re-visit this again, but is it because they (_both_) belong to another production, or because at least one of them already belongs to another production? If it's the latter, the method name and message might be confusing. If it's the first one it's all good 👍

Reply:
It's indeed the latter, but it could be both the events. It's hard to know with the current flow which one is the problem. Do you think we should make the exception message and method name even more explicit?

Reply:
I'd suggest making it more descriptive but not include whether it's specifically one or both, because I can see why that would be hard yes. So something like `becauseOneOrBothAlreadyBelongToAnotherProduction`? Or `becauseTheyOverlapWithAnotherProduction`? Just some ideas to give you inspiration, doesn't have to be either if you can think of something else that you like better.

But at this point it's the last 1% so if you disagree or want to move on, feel free to just merge.

Reply:
Was already fixing it and I think I found a clean description using `some`